lib/request.js

```diff
@@ -233,11 +242,25 @@ Request.prototype.sendResponse = function(response) {
 
   response.writeHead(this.response.statusCode, headers);
 
-  if (this.response.filePath) {
-    var readStream = fs.createReadStream(this.response.filePath);
-    readStream.pipe(response);
+  if (isStream(this.response.body)) {
+    // Because we may need to respond with this body more than once, if it is a stream,
+    // we make a buffer copy and use that as the body for future responses.
```

Comment:
I'm a little concerned about the implication here if the Stream is massive in size; the copy could easily saturate the memory of a small VM. Obviously Hock is used predominantly for testing, but it's still a concern.

It's possible we could simply not support a stream as the body whenever `Request._maxRequests > 1`.

Thoughts?